backend/src/settings.py

```diff
@@ -23,9 +23,9 @@
 SECRET_KEY = 'raeq^$)2(2mt)1s*o_i6vb!chw^9#3uri02(r7kn=_xs7d(08%'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
Before we push to our main, can we set debug to false? That way we dont open our debug ports on AWS. Jenkins will always deploy the latest code in the main branch.